dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/programrule/hibernate/ProgramRuleAction.hbm.xml

```diff
@@ -43,6 +43,9 @@
 
     <property name="templateUid" column="templateuid"/>
```

Comment:
Why do we need to keep templateUid column?

Reply:
its still in the ProgramRuleAction. It will be removed after 2.42.